third_party/terraform/utils/provider.go.erb

```diff
@@ -98,12 +97,45 @@ func Provider() terraform.ResourceProvider {
 			"google_project_services":                dataSourceGoogleProjectServices(),
 			"google_service_account":                 dataSourceGoogleServiceAccount(),
 			"google_service_account_key":             dataSourceGoogleServiceAccountKey(),
-			"google_storage_bucket_object":           dataSourceGoogleStorageBucketObject(),
 			"google_storage_object_signed_url":       dataSourceGoogleSignedUrl(),
 			"google_storage_project_service_account": dataSourceGoogleStorageProjectServiceAccount(),
 		},
 
-		ResourcesMap: mergeResourceMaps(
+		ResourcesMap: ResourceMap(),
+
+		ConfigureFunc: providerConfigure,
+	}
+}
+
+func mergeResourceMaps(ms ...map[string]*schema.Resource) (map[string]*schema.Resource, error) {
```

Comment:
I have a weak preference to leave this code in utils rather then pulling into provider.